app/settings.html

```diff
@@ -23,13 +19,9 @@
       </p>
       <h2>breaks</h2>
       <p>
-        <span id="breakDurationMinus" class="pointer">-</span>
-        <span id="breakDuration"></span>
-        <span id="breakDurationPlus" class="pointer">+</span>
+        <input type="number" id="breakDuration" min="0" max="59" step="1" value=5>
```

Comment:
Did you check how it looks? It's different after your changes. 

<img width="798" alt="screen shot 2017-03-23 at 21 13 07" src="https://cloud.githubusercontent.com/assets/4745859/24268325/b2dc1618-100d-11e7-91ee-f744e07867e8.png">

Also step should not be 1. Check how previous version was. 

That is why I am doing it how I am doing it. It looks better and I do not need to write css for inputs.